src/shared/components/header/header.component.scss

```diff
@@ -1,73 +1,101 @@
-.page-header {
-  display: flex;
-  flex-direction: row;
-  justify-content: space-between;
-  align-items: center;
-  height: 100%;
-  padding: 0 24px 0 24px;
-  height: 56px;
-  position: relative;
+ion-toolbar {
+  --padding-bottom: 0px;
+  --padding-start: 0px;
+  --padding-end: 0px;
+  --padding-top: 0px;
+  --background: #0d0d0d;
 
-  .logo-container {
+  .page-header {
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    align-items: center;
+    height: 100%;
```

Comment:
Why two heights? One with % and one with px?

Reply:
mistake. fixed.